File: 1098 - Sequencia IJ 4/sequencia_ij_4.py
for step in range(11):
    i = step * 0.2
    for k in range(3):
        j = 1 + i + k
        if step % 5 == 0:
            print(f"I={int(round(i))} J={int(round(j))}")
        else:
            print(f"I={i:.1f} J={j:.1f}")

File: 1098 - Sequencia IJ 4/sequencia_ij_4.c
#include <stdio.h>
#include <math.h>

int main(){
    for(int step = 0; step <= 10; step++){
        double i = step * 0.2;
        for(int k = 0; k < 3; k++){
            double j = 1 + i + k;
            if(step % 5 == 0)
                printf("I=%d J=%d\n", (int) round(i), (int) round(j));
            else
                printf("I=%.1f J=%.1f\n", i, j);
        }
    }
    return 0;
}
